src/vorta/views/repo_add_dialog.py

```diff
@@ -246,3 +219,13 @@ def run(self):
                 self.thread.run()
             else:
                 self._set_status(params['message'])
+
+    @property
+    def values(self):
+        out = dict(
+            ssh_key=self.sshComboBox.currentData(),
+            repo_url=self.repoURL.text(),
+            password=self.passwordInput.get_password(),
+            extra_borg_arguments=self.extraBorgArgumentsLineEdit.text(),
+        )
+        return out
```

Comment:
This could be moved to the superclass because it's common to both subclasses.

Reply:
I added them to subclass because of difference in dict items. `ExistingRepoWindow` does not use `encryption` value as there's no such widget. Would it more sense to include it conditionally using a private variable?

Reply:
You can override the method in the subclass but still leverage the super class implementation through `super().values()`.

Reply:
Thank you, that was helpful.